app/models/class_import.rb

```diff
@@ -62,5 +64,14 @@ def postprocess_rows!
       .patient_sessions
       .where(patient: unknown_patients)
       .find_each(&:destroy_if_safe!)
+
+    # Add the unknown patients to the generic clinic
```

Comment:
Don't we want this to happen only when the change from <specific school> => <school unknown> has been confirmed via the UI?

Reply:
Yeah, we will want that. For now I made it automatic as with everything else because that UI hasn't been built yet, under the assumption that it would be changed to use that UI in the future.